AppiumForMac/Server/Controller/AfMSessionController.m

```diff
@@ -574,6 +574,26 @@ -(void) closeWindow
     [self.currentWindow performAction:@"AXCancel"];
 }
 
+-(NSString*)executeShellScript:(NSString*)script
+{
+    FILE *fp;
+    const int BUFFSIZE = 512;
+    char buffer[BUFFSIZE];
+    const char *command = [script UTF8String];
+    NSMutableString *output = [[NSMutableString alloc] initWithString:@""];
+    fp = popen(command, "r");
+    if (!fp){
+        NSLog(@"Open pipe failed\n");
+        return @"";
+    }
+    while (fgets(buffer, BUFFSIZE, fp)!=NULL)
+    {
```

Comment:
`while xxxxx {` style like the above `if` or change the `if (...) {` to this `while`?